src/contexts/UserDataProvider.js

```diff
@@ -0,0 +1,49 @@
+import React, { createContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  USERS_API_ROUTE,
+  CURRENT_USER_PARAMETER,
+  buildUsersEndpoint,
+  buildApiOptions,
+} from '../api/graasp';
+
+export const UserDataContext = createContext();
+
+const UserDataProvider = ({ children }) => {
+  const [userId, setUserId] = useState();
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      const baseUrl = process.env.REACT_APP_BASE_URL;
```

Comment:
Get this from `config/env.js` not directly in here.

Reply:
Just to clarify, is the suggestion to do `export const baseUrl = process.env.REACT_APP_BASE_URL` in `config/env.js`, and then reference *that* `baseUrl` in these Contexts (note that `SpaceDataProvider` and `TaskDataProvider` also currently reference `process.env.REACT_APP_BASE_URL` directly)?

Reply:
Something like:

```
// config/env.js
export const {
  ...
  REACT_APP_BASE_URL,
} = process.env;
```

```
// usage
import { REACT_APP_BASE_URL } from '../../config/env.js'
...
```

Reply:
👍